engine/flink/process/src/test/scala/pl/touk/nussknacker/engine/process/helpers/SampleNodes.scala

```diff
@@ -191,6 +191,23 @@ object SampleNodes {
 
   }
 
+  object CollectingEagerService extends EagerService {
+
+    @MethodToInvoke
+    def invoke(@ParamName("static") static: String,
```

Comment:
Can you do declaring invoke in one line? :)

Reply:
ok...